base/test.jl

```diff
@@ -368,6 +368,7 @@ type TestSetException <: Exception
     fail::Int
     error::Int
     broken::Int
+    errors_and_fails
```

Comment:
and I guess some dummy cases where you're sending `nothing` could be replaced by empty arrays? this field holds an array of something, is the element type always the same and known?